dnf/cli/commands/check.py

```diff
@@ -68,7 +68,7 @@ def configure(self):
             self.opts.check_types = {'all'}
         else:
             self.opts.check_types = set(self.opts.check_types)
-        self.base.conf.disable_excludes += ("all",)
+        self.base.conf.disable_excludes += ["all"]
```

Comment:
it would actually be good to ensure that doing `+= (tuple)` continues to work, because that's the one way of appending to one of these options I've found that works in every DNF version so far, including 3.6.